### DISCUSSION TITLE
Wrong Date Displaying for Multi Date Event

### DISCUSSION THREAD

**Discussion by mrjeffbarnes** [add2cal/add-to-calendar-button#256] [Q&A] [answered]
I have created a multi-date event with three dates over the course of a week, and everything works properly on the calendar side, but on the display of the drop downs for the dates, the wrong date is showing. I can only assume this has something to do with time zones, but I'm not sure how to fix it.

Here is what we have:

```
<div class="atcb" style="display:none;" size="8|6|4" align="center">
{
  "name":"Event Name",
  "dates":[
    {
      "name":"Name",
      "description":"Description",
      "startDate":"2022-12-08",
      "startTime":"09:00",
      "endTime":"10:00"
    },
    {
      "name":"Name",
      "description":"description",
      "startDate":"2022-12-08",
      "startTime":"16:00",
      "endTime":"17:00"
    },
  {
      "name":"Event",
      "description":"Description",
      "startDate":"2022-12-15",
      "startTime":"16:00",
      "endTime":"17:00"
    }
  ],
  "location":"Location",
  "options":[
    "Apple",
    "Google",
    "iCal",
    "Outlook.com",
    "Yahoo"
  ],
  "timeZone":"America/Los_Angeles",
  "iCalFileName":"Multi-Date-Event"
}
</div>
```

but when I am testing it, this is what we see for the drop down of dates. All the times are correct, the name, location, etc. all correct, but the display on the left side is one day early for each event, and I'm not sure what I'm doing wrong here. 

<img width="376" alt="Screen Shot 2022-11-27 at 7 04 23 AM" src="https://user-images.githubusercontent.com/119197957/204142261-3590cfbf-dc73-492d-8e65-7e3933c1677a.png">

**Comment by jekuer** [comment, marked as answer]
Thanks for bringing this up!
This is a clear bug. Should be fixed with the latest patch v1.18.8

**Reply by mrjeffbarnes** [reply]
thank you @jekuer. Any idea when that is planned to be release? Just wondering if I should plan on using this code still now or not since we're trying to promote for next week.  Thanks!

**Reply by jekuer** [reply]
It already got released 😉
